marklogic-data-hub-central/ui/src/components/header/system-info.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {render, cleanup, } from "@testing-library/react";
+import SystemInfo from './system-info';
+import {AuthoritiesContext, AuthoritiesService} from "../../util/authorities";
+import {BrowserRouter as Router} from "react-router-dom";
+import data from '../../assets/mock-data/system-info.data';
+
+describe('Update data load settings component', () => {
+
+    afterEach(() => {
+        cleanup();
+    });
+
+    test('verify project info display, user with "Download" button enabled', async () => {
+        const authorityService = new AuthoritiesService();
+        authorityService.setAuthorities(['downloadProjectFiles']);
+        const { getByText } = render(<Router><AuthoritiesContext.Provider value={authorityService}>
+            <SystemInfo 
+                serviceName={data.environment.serviceName}
+                dataHubVersion={data.environment.dataHubVersion}
+                marklogicVersion={data.environment.marklogicVersion}
+                systemInfoVisible={true} 
+                setSystemInfoVisible={jest.fn()}
```

Comment:
This is good. I would probably add systemInfoVisible: true, setSystemInfoVisible: jest.fn() in system-info.data.ts and use that here like so: {...data.environment}

Reply:
@bsrikan:

> Can we also pls update "marklogic-data-hub-central/ui/e2e/cypress/support/commands.js" to include systemInfo

Can you do this? I haven't done anything with the e2e cypress stuff, not sure what you're looking for.

Reply:
@bsrikan:

> This is good. I would probably add systemInfoVisible: true, setSystemInfoVisible: jest.fn() in system-info.data.ts and use that here like so: {...data.environment}

Would be nice for brevity, but I don't want to put those additional properties in the environment object since they aren't returned from the environment method in the app.

Reply:
Sure @wooldridge makes sense for test data props. I will update the endpoint for the cypress command support with the new endpoint.